src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs

```diff
@@ -3500,7 +3501,10 @@ private IEnumerable<PSModuleInfo> CreateFakeModuleObject(IEnumerable<ModuleSpeci
                             }
 
                             ss.Internal.ExportedVariables.Clear();
-                            ss.Internal.ExportedVariables.AddRange(updated);
+                            foreach(var psVariable in updated)
+                            {
+                                ss.Internal.ExportedVariables.Add(psVariable);
```

Comment:
`ConcurrentBag` does not have the `AddRange` API, hence this change. There are only 2 places where I have to do the `foreach` 'by hand', therefore I do not think it is worthwhile creating an `AddRange` extension method unless you feel otherwise